### ISSUE TITLE
Failed to read schema document `https://www.eolang.org/xsd/XMIR-0.46.0.xsd`

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#132]
Recently [one of the pipelines](https://github.com/objectionary/jeo-maven-plugin/actions/runs/12258856491/job/34199619501?pr=942) in `jeo-maven-plugin` failed with the following error: 

```xml
-1:-1: schema_reference.4: Failed to read schema document 'https://www.eolang.org/xsd/XMIR-0.46.0.xsd', because 1) could not find the document; 2) the document could not be read; 3) the root element of the document is not <xsd:schema>.
```
[build.log](https://github.com/user-attachments/files/18092081/build.log)

Moreover, it failed only on `ubuntu` system. On `macos` everything is fine.

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you take a look, please? This issue blocks any progress in `jeo-maven-plugin`.

**Comment by volodya-lombrozo** [comment]
Where it happened:
```java
    at com.jcabi.xml.StrictXML.<init> (StrictXML.java:122)
    at com.jcabi.xml.StrictXML.<init> (StrictXML.java:93)
    at com.jcabi.xml.StrictXML.<init> (StrictXML.java:84)
    at com.jcabi.xml.StrictXML.<init> (StrictXML.java:74)
    at org.eolang.jeo.representation.VerifiedEo.asXml (VerifiedEo.java:71)
```

**Comment by yegor256** [comment]
@volodya-lombrozo the URL is valid. The problem most probably is related to some instability of network connection. The best we can do is to introduce a retrying mechanism to `StrictXML`, but I'm not sure how exactly...

**Comment by volodya-lombrozo** [comment]
@yegor256 I believe we shouldn't rely on an internet connection here. It creates an enormous number of bugs and flaky tests. It's better to keep everything local. In other words, now we can't develop without the internet. Btw, it is often the case.

As you can see, I faced with this problem immediately.

**Comment by volodya-lombrozo** [comment]
@yegor256 btw, it might be interesting https://stackoverflow.com/questions/28267536/how-to-refer-to-a-classpath-xsd-in-xsischemalocation
